Documentation/Helm-Charts/operator-chart.md

```diff
@@ -137,6 +137,7 @@ The following table lists the configurable parameters of the rook-operator chart
 | `csi.topology.enabled` | Enable topology based provisioning | `false` |
 | `currentNamespaceOnly` | Whether the operator should watch cluster CRD in its own namespace or not | `false` |
 | `disableDeviceHotplug` | Disable automatic orchestration when new devices are discovered. | `false` |
+| `discover.customHostnameLabel` | Custom label to identify node hostname. If not set `kubernetes.io/hostname` will be used | `nil` |
```

Comment:
what if someone wants to use both `kubernetes.io/hostname` and a new label,

can we have a generic setting for custom list of names and make the default one always true??

Reply:
Agreed, let's not try to support fallback. Either we use the current label, or a custom one, but not both.